CHANGELOG.md

```diff
@@ -24,8 +30,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - A configurable variable `cluster_cookie` for `tt create cartridge` template.
 - ``tt build`` accepts application name for building.
-- Creating wal, vinyl and memtx directories for `tt pack`. If these directories 
-  are not located in the same directory in the environment for packing, the result package 
+- Creating wal, vinyl and memtx directories for `tt pack`. If these directories
```

Comment:
This changes doesn't belong to the commit.

Reply:
Right. Got into it again.